src/import-v1/index.ts

```diff
@@ -220,7 +221,7 @@ class SubdomainTransform extends stream.Transform {
         zonefile: '',
         parent_zonefile_hash: parts[1],
         fully_qualified_subdomain: parts[2],
-        owner: parts[3],
+        owner: c32check.b58ToC32(parts[3]), //convert btc address to stx,
```

Comment:
Is this also needed on the addresses for `names` and `namespaces`?

Reply:
No!, they are already in Stx as only subdomains were being registered from `subdomain-registrar` which was previously dealing in Btc address.